src/examples/dac/dac.go

```diff
@@ -0,0 +1,40 @@
+// Simplistic example using the DAC on the Circuit Playground Express.
+//
+// To actually use the DAC for producing complex waveforms or samples requires a DMA
+// timer-based playback mechanism which is beyond the scope of this example.
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+func main() {
+	enable := machine.PA30
+	enable.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	enable.Set(true)
+
+	speaker := machine.A0
+	speaker.Configure(machine.PinConfig{Mode: machine.PinOutput})
+
+	machine.DAC0.Configure(machine.DACConfig{})
+
+	data := []uint16{4096, 2058, 1024, 512, 256, 128}
```

Comment:
Since samd21 is a 10-bit DAC, numbers above 1024 are meaningless.
But I want to use the same test with samd51, so I think it is better to leave it as it is.

Samd51 is a 12-bit DAC and its maximum value is 4095.
In addition, since 2058 also looks wrong, isn't 2048 the correct value?

In other words, I think the following is correct.

```go
data := []uint16{4095, 2048, 1024, 512, 256, 128}
```